applications/launcher/controller.cpp

```diff
@@ -216,7 +216,6 @@ QList<QObject*> Controller::getApps(){
 void Controller::setAutoStartApplication(QString autoStartApplication){
     m_autoStartApplication = autoStartApplication;
     emit autoStartApplicationChanged(autoStartApplication);
-    saveSettings();
```

Comment:
Was this intended to be left it? I see that you are still calling `controller.saveSettings()` elsewhere?

Reply:
It needs to be removed otherwise loadSettings calls saveSettings via setProperty->setAutoStartApplication->saveSettings.

Reply:
Fair enough. You wouldn't believe how many hours I've spent already trying to track down what was causing this in the past. I'm glad you managed to figure it out!